### PR TITLE
German chat translator

### PR DESCRIPTION
**Pull request by sbarrack**
With this setting enabled, playing on the German server will (attempt to) translate other people's chat messages to English. It has a clear visual indication when each message is translated and uses free, unlimited APIs on public servers.

It will not translate messages to English when you're playing on the English server, even if the setting is enabled. That way Americans can leave it on all the time :)

(`master` branch on my fork is `oldSkribbl` on the base repository.)

### CODE REVIEW

File: popup/popup.html [hunk 1]
```diff
@@ -59,24 +59,27 @@ <h1>Typo Dashboard</h1>
...
                 <div class="flexcol">
                     <div class="flexrow flexrowMenu">
-                        <button type="button" id="imageagent">ImageAgent</button>
-                        <button type="button" id="markup">Markup</button>
-                        <button type="button" id="keybinds">Keybinds</button>
+                        <button type="button" id="translate">Translate (DE &rarr; EN)</button>
                     </div>
                     <div class="flexrow flexrowMenu">
-                        <button type="button" id="clearcanvas">Don't Clear</button>
+                        <button type="button" id="keybinds">Keybinds</button>
```
Comment (toobeeh) [inline popup/popup.html:65]: Please provide a screenshot of the new popup layout.

**Review by toobeeh — APPROVED** [review]
Cool stuff! 
Good job on digging through the mess on how commands and features work in the popup and command handler.

**Comment by toobeeh** [comment]
Thinking about it, maybe the translation toggle would better fit to the "advanced" tab.

If you ever would add other languages to translate, a selection menu would find the necessary space there, too.
For now, i think i'd approve it as-it-is, but if you ever plan to extend the feature please move it to the advanced tab (or i'll do it at some point myself, idk)

**Comment by sbarrack** [comment]
New popup  

a

File: popup/popup.html (hunk 1 above)
Comment (sbarrack) [reply popup/popup.html:65]: New popup  

a

**Comment by toobeeh** [comment]
Ah, it really is a bit prominent considering the fact that it's relevant for only about 50% of the users.
If you don't mind, please just move the button to the advanced tab & the PR will be ready to merge.

**Pull request merged**